From 6b75c487808600236d10bdca6ecfe7cd6506a334 Mon Sep 17 00:00:00 2001
Date: Sat, 6 Oct 2018 00:08:50 +0200
Subject: [PATCH] cmake: allow user to override CMAKE_DEBUG_POSTFIX

Default is unchanged, but this way the postfix can be set to "" if
wanted.
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 0fe939df..74bda282 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -60,7 +60,7 @@ endif()
 check_include_file(unistd.h Z_HAVE_UNISTD_H)
 
 if(MSVC)
-    set(CMAKE_DEBUG_POSTFIX "d")
+    set(CMAKE_DEBUG_POSTFIX "d" CACHE STRING "Set debug library postfix")
     add_definitions(-D_CRT_SECURE_NO_DEPRECATE)
     add_definitions(-D_CRT_NONSTDC_NO_DEPRECATE)
     include_directories(${CMAKE_CURRENT_SOURCE_DIR})
